packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -53,6 +55,35 @@ const Tolerance _kFlingTolerance = const Tolerance(
 /// in the context of tests. In other contexts, you will have to either pass a [TickerProvider] from
 /// a higher level (e.g. indirectly from a [State] that mixes in [TickerProviderStateMixin]), or
 /// create a custom [TickerProvider] subclass.
+///
+/// The methods that start animations return a [TickerFuture] object which
+/// completes when the animation completes successfully, and never throws an
+/// error; if the animation is aborted, the future never completes. This object
```

Comment:
replaced aborted with canceled throughout.
for the stop() argument, changed completed to canceled and reversed the semantic.